Sources/EpoxyCore/Views/PlatformView.swift

```diff
@@ -0,0 +1,31 @@
+// Created by Cal Stephens on 6/26/23.
+// Copyright © 2023 Airbnb Inc. All rights reserved.
+
+import SwiftUI
+
+#if canImport(UIKit)
+import UIKit
+
+public typealias UIViewOrNSView = UIView
+public typealias UIViewControllerOrNSViewController = UIViewController
```

Comment:
```suggestion
public typealias ViewControllerType = UIViewController
```

overall think naming like this might be nicer and less verbose.